flow.py

```diff
@@ -22,6 +24,7 @@ def __init__(self, identifier, source, destination, amount, start_time):
         self.event_scheduler = None
         self.logger = None
         self.complete = False
+        self.reno = CongestionControlReno()
```

Comment:
The Flow should have a property `controller` or something, not `reno`, I think. I think it'd make sense for the controller to be passed in as a parameter of the init function. (It's be reasonable to put an assertion that the passed in argument is a CongestionController class.)